lib/book.py

```diff
@@ -443,28 +281,27 @@ def create_xray(self, aConnection, sConnection, info, log=None, abort=None, remo
             return
         if log: log('%s \t\tCreating x-ray...' % datetime.now().strftime('%m-%d-%Y %H:%M:%S'))
         self._write_xray(info, remove_files_from_dir=remove_files_from_dir)
-        return (aConnection, sConnection)
+        return connection
```

Comment:
I tested this today. Looks like the original connection will not be updated if the new one isn't returned.